Gemfile

```diff
@@ -1,13 +1,15 @@
 source 'https://rubygems.org'
 
-gem 'jbuilder',          '~> 2.0'
-gem 'json-schema',       '~> 2.8'
-gem 'manageiq-loggers',  '~> 0.1'
-gem 'manageiq-password', '~> 0.1'
-gem 'pg',                '~> 1.0', :require => false
-gem 'puma',              '~> 3.0'
-gem 'rack-cors',         '>= 0.4.1'
-gem 'rails',             '~> 5.2.2'
+gem 'jbuilder',           '~> 2.0'
+gem 'json-schema',        '~> 2.8'
+gem 'manageiq-loggers',   '~> 0.1'
+gem 'manageiq-messaging', '~> 0.1.2'
+gem 'manageiq-password',  '~> 0.1'
+gem 'pg',                 '~> 1.0', :require => false
+gem 'prometheus-client',  '~> 0.8.0'
```

Comment:
Is this used?

Reply:
I was actually going to ask, I don't think so? It got pulled in during a merge conflict. I think it can be removed.